setup.py

```diff
@@ -56,16 +56,21 @@ def parse_arg_remove_string(argv, arg_name_equal):
 
 cuda_version = None
 rocm_version = None
+is_migraphx = False
 is_rocm = False
 is_openvino = False
 # The following arguments are mutually exclusive
 if wheel_name_suffix == "gpu":
     # TODO: how to support multiple CUDA versions?
     cuda_version = parse_arg_remove_string(sys.argv, "--cuda_version=")
 elif parse_arg_remove_boolean(sys.argv, "--use_rocm"):
+    is_migraphx = parse_arg_remove_boolean(sys.argv, "--use_migraphx")
```

Comment:
Is the intent here that if "--use_rocm --use_migraphx" is specified in that order, then the package will be called "rocm"?  What if the order is swapped "--use_migraphx --use_rocm"?

Reply:
That was the intention. I changed to `onnxruntime-rocm` for both EPs.